tests/apps/crunchy/test_crunchy.py

```diff
@@ -107,14 +111,38 @@ def test_is_fastq_compression_possible_spring_exists(
     assert spring_file.exists()
 
     # WHEN checking if SPRING compression is done
-    result = crunchy_api.is_fastq_compression_possible(compression_object)
+    result = crunchy_api.is_fastq_compression_possible(
+        compression_obj=compression_object, is_spring_archived=False
+    )
 
     # THEN result should be False since the compression flag exists
     assert result is False
     # THEN assert that the correct message was communicated
     assert "SPRING file found" in caplog.text
 
 
+def test_is_fastq_compression_possible_spring_is_archived(
+    crunchy_config: dict[str, dict[str, Any]], compression_object: CompressionData, caplog
+):
+    """Test if FASTQ compression is possible when the Spring file is archived
+
+    This means that the SPRING file exists
+    """
+    caplog.set_level(logging.DEBUG)
+    # GIVEN a crunchy-api, and existing FASTQ paths
+    crunchy_api = CrunchyAPI(crunchy_config)
```

Comment:
Isn't there a fixture with the API already?

Reply:
if there isn't, consider creating it if you think it fits in the scope of this PR

Reply:
There was not one that was reachable. I created one, but quite possibly there should be a refactoring PR removing the fixture in some more specialized directories.